index.html

```diff
@@ -94,26 +120,55 @@ <h3><a class="textdecoration font1" href="#">Tonic</a></h3>
 
             </div>
 
+            <div class="article1div show2">
+
+                <h3><a class="textdecoration font1" href="#">Tonic</a></h3>
+
+                <ul class="ul1">
+
+                    <li><a class="textdecoration font2" href="#">CANOPY</a></li>
+                    <li><a class="textdecoration" href="#"><img src="./img/mid_dot.jpg" alt=" "></a></li>
+                    <li><a class="textdecoration font3" href="#">Back End Dev</a></li>
+                    <li><a class="textdecoration" href="#"><img src="./img/mid_dot.jpg" alt=" "></a></li>
+                    <li><a class="textdecoration font3" href="#">2015</a></li>
+
+                </ul>
+
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+
+                <ul class="gridbox1">
+
+                    <li class="background1"><a class="textdecoration font4" href="#">html</a></li>
+                    <li class="background1"><a class="textdecoration font4" href="#">Ruby on rails</a></li>
+                    <li class="background1"><a class="textdecoration font4" href="#">css</a></li>
+                    <li class="background1"><a class="textdecoration font4" href="#">javaScript</a></li>
+
+                </ul>
+
+                <button type="submit"><a class="font5 textdecoration" href="#">See Project</a></button>
+
+            </div>
```

Comment:
- [ ] Good job alternating the image and project description. I think there's a little more to do to match the Figma design.
Kindly check the whitespace to the left and right of the component. 😎️

- Your Design
![Screenshot from 2022-12-08 20-26-00](https://user-images.githubusercontent.com/17243325/206531389-4d42cee6-72c2-4c66-b4d0-d51d0814a45e.png)

- Figma Design
![Screenshot from 2022-12-08 20-25-04](https://user-images.githubusercontent.com/17243325/206531604-93ca0d01-6e94-4db4-800e-166f9f9310db.png)